src/server/storage_engine/default_engine.rs

```diff
@@ -69,6 +71,7 @@ impl StorageEngine for DefaultEngine {
             Database { db, db_opts, path }
         };
 
+        println!("root: {}", root);
```

Comment:
use log

Reply:
This is for debug about unit test. Already removed.